paddle/fluid/operators/CMakeLists.txt

```diff
@@ -184,8 +184,8 @@ else()
     set(DEPS_OPS ${DEPS_OPS} nccl_op)
 endif()
 
-add_subdirectory(detail)
 if(WITH_DISTRIBUTE)
+    add_subdirectory(distributed)
 
     set(DISTRIBUTE_DEPS "")
     if(WITH_GRPC)
```

Comment:
how about use a common cmake function to package
```
op_library(prefetch_op DEPS ${DISTRIBUTE_DEPS})
set_source_files_properties(prefetch_op.cc PROPERTIES COMPILE_FLAGS ${DISTRIBUTE_COMPILE_FLAGS})	     
```
to simply line 198-209

Reply:
Thanks, done.